src/qt/forms/createwalletdialog.ui

```diff
@@ -62,64 +65,77 @@
     </rect>
    </property>
    <property name="toolTip">
-    <string>Encrypt the wallet. The wallet will be encrypted with a passphrase of your choice.</string>
+    <string>Private keys in the wallet will be encrypted with a passphrase of your choice. Addresses, transaction history and notes are not encrypted.</string>
    </property>
    <property name="text">
-    <string>Encrypt Wallet</string>
+    <string>Encrypt private keys</string>
    </property>
    <property name="checked">
-    <bool>true</bool>
+    <bool>false</bool>
+   </property>
+  </widget>
+  <widget class="QLabel" name="encrypt_wallet_label">
+   <property name="geometry">
+    <rect>
+     <x>20</x>
+     <y>90</y>
+     <width>130</width>
+     <height>21</height>
+    </rect>
+   </property>
+   <property name="text">
+    <string>Advanced options</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="disable_privkeys_checkbox">
    <property name="enabled">
-    <bool>false</bool>
+    <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
-     <y>80</y>
+     <y>115</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
-    <string>Disable private keys for this wallet. Wallets with private keys disabled will have no private keys and cannot have an HD seed or imported private keys. This is ideal for watch-only wallets.</string>
+    <string>Disable private keys for this wallet. Watch-only wallets have no private keys and cannot have an HD seed or imported private keys.</string>
    </property>
    <property name="text">
-    <string>Disable Private Keys</string>
+    <string>Watch-only</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="blank_wallet_checkbox">
    <property name="geometry">
     <rect>
      <x>20</x>
-     <y>110</y>
-     <width>171</width>
+     <y>135</y>
+     <width>220</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
-    <string>Make a blank wallet. Blank wallets do not initially have private keys or scripts. Private keys and addresses can be imported, or an HD seed can be set, at a later time.</string>
+    <string>Blank wallets do not initially have private keys or scripts. Private keys and addresses can be imported, or an HD seed can be set, at a later time.</string>
    </property>
    <property name="text">
-    <string>Make Blank Wallet</string>
+    <string>Manually import seed or keys</string>
```

Comment:
Unless the manual import takes place immediately here on creation, this change seems confusing; "Blank wallet", along with the previous tooltip, seems more clear.

Reply:
Anyone using this feature will probably be following a tutorial, so I don't think we should bikeshed this to death. I can see a followup design improvement adding a wizard where you can copy-paste descriptors at this step.

Reply:
P.S. apart from this ^, I'll be very reactive in feedback/review here to try to move it forward.

Reply:
Fair enough, but this change seems strictly worse, as it's really misleading (and wasn't in your original proposal), nothing is being imported at this step.

Reply:
Maybe either:

Make blank wallet / tootip: Make a blank wallet. Blank wallets do not initially have private keys or scripts. Private keys and addresses can be imported, or an HD seed can be set, at a later time.

or

Skip seed generation / tooltip: Skip seed generation and make a blank wallet without private keys or scripts. Private keys and addresses can be imported, or an HD seed can be set, at a later time.

Reply:
I'll test the rest.